webtask.json

```diff
@@ -36,6 +36,22 @@
       "description": "A CSS file containing custom styles for the extension",
       "example": "https://cdn.fabrikam.com/static/extensions/theme/fabrikam.css",
       "required": false
+    },
+    "ALLOW_AUTHZ": {
+      "description": "This setting allow to enable Authorization Extension integration (Deprecated)",
```

Comment:
Let's just shorten this description to `Enable Authorization Extension integration (Deprecated)`

Reply:
That's not quite true. `ALLOW_AUTHZ` **allows** user to enable integration. To actually enable it, user should go to settings in the extension itself, and enable it there.

Reply:
Ok, let's make it `Allow enabling of Authorization Extension integration (Deprecated)`